src/ImpactEvaluator.sol

```diff
@@ -7,18 +7,19 @@ contract ImpactEvaluator is AccessControl {
     struct Round {
         uint start;
         string[] measurementCids;
-        address[] participantAddresses;
+        address payable[] participantAddresses;
         uint[] participantScores;
         bool scoresSubmitted;
         string summaryText;
     }
 
     Round[] public rounds;
     uint public roundLength = 10;
+    uint public roundReward = 100;
```

Comment:
I think we should have a getter for this and an setter function that is restricted to the admin. Thoughts?

Reply:
iiuc a public value automatically has a getter, and I have an implementation of the admin restricted setter below